code/game/objects/effects/proximity.dm

```diff
@@ -1,15 +1,16 @@
 /datum/proximity_monitor
-	var/atom/host //the atom we are tracking
-	var/atom/hasprox_receiver //the atom that will receive HasProximity calls.
-	var/atom/last_host_loc
-	var/list/checkers //list of /obj/effect/abstract/proximity_checkers
+	///The atom we are tracking
+	var/atom/host
+	///The atom that will receive HasProximity calls.
+	var/atom/hasprox_receiver
 	var/current_range
-	var/ignore_if_not_on_turf //don't check turfs in range if the host's loc isn't a turf
-	var/wire = FALSE
+	///If we don't check turfs in range if the host's loc isn't a turf
+	var/ignore_if_not_on_turf
+	var/list/loc_connections = list(
```

Comment:
Sorry?

Reply:
I ask cause doing lists in the object def like this can increase initialization costs semi significantly. If we don't init em often tho it's not a huge issue

Reply:
Also can you please not mark as resolved reviews that you reply to? S fine for the ones where you just implement a change, but it's hard to have a conversation if I can't find it in the first place.

Reply:
Well, loc connections is now a static list so this shouldn't be an issue anymore.